modules/test/playwright/tests/export-import-web/pages/CompanyExportImportPage.ts

```diff
@@ -116,14 +116,21 @@ export class CompanyExportImportPage {
 
 	async import(
 		filePath: string,
-		includePermissions: boolean = false
+		includePermissions: boolean = false,
+		expectedErrorMessage?: string
 	): Promise<void> {
 		await this.applicationsMenuPage.goToImport();
 
 		await this.exportImportPage.newImportButton.click();
 
 		await this.page.locator('input[type="file"]').setInputFiles(filePath);
 
+		if (expectedErrorMessage !== null) {
+			expect(this.page.getByText(expectedErrorMessage)).toBeVisible();
```

Comment:
Is the await instruction necessary here?

```
await expect(this.page.getByText(expectedErrorMessage)).toBeVisible();
```

I see that pattern in other files